srcpkgs/paleta/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'paleta'
+pkgname=paleta
+version=1.0.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config tar"
+makedepends="cmake"
```

Comment:
This project doesn't use `cmake`, it's not needed here. `build_style=gnu-makefile` will `make` and `make install` with right CFLAGS.

Reply:
Right, kinda just starting to make sense to me in its entirety lmao. Will get back in a bit